Assets/Code/src/Runtime/Match/Components/MatchManager.cs

```diff
@@ -15,6 +15,7 @@ public class MatchManager : MonoBehaviour {
 
   public bool IsLocal => Config.IsLocal;
   public bool IsPaused { get; private set; }
+  public bool isControllerLocked { get; private set; }
```

Comment:
Style: Please use uppercase properties for public properties. 

Might also be a good idea to also inverse this: IsControllerRunning, which defaults to false, set only in the RunMatch method.